tests/tools/DEBUG_MultiMeshManager.cpp

```diff
@@ -0,0 +1,96 @@
+#include "DEBUG_MultiMeshManager.hpp"
+#include <catch2/catch_test_macros.hpp>
+#include <wmtk/Mesh.hpp>
+#include <wmtk/multimesh/utils/tuple_map_attribute_io.hpp>
+#include "DEBUG_Mesh.hpp"
+
+
+namespace wmtk::tests {
+void DEBUG_MultiMeshManager::check_child_mesh_valid(const Mesh& my_mesh, const Mesh& child_mesh)
```

Comment:
This should perform some checks in the future, right? In that case I would prefer the "throw a message" thing and a `CHECK(false)` to make sure that other people are not using it without being aware that it does not do anything.

Reply:
i dont know - i can't remember who/what the original purpose was for this and just copied it over. tired :(